src/blog/2024/04/node-red-dashboard-2-layout-navigation-styling.md

```diff
@@ -0,0 +1,186 @@
+
+#### Fixed layout
+
+In this layout, the horizontal (width) unit in the size property is fixed at 90px. For example, if you set the group width to 3 units, it will be 3 * 90 = 270px wide. Each widget size unit represents a column in the group's internal grid, following the same pattern as other layouts.
+
+!["Screenshot of dashboard having fixed layout"](./images/node-red-dashboard-2-layout-navigation-styling-fix-layout.png "Screenshot of dashboard having fixed layout"){data-zoomable}
+
+*Note: Currently this layout is not completely ready to use so it is recommended to use other layouts.*
+
+### Setting page layout
+
+1. Navigate to the page configuration by clicking on the **edit** button of your page in the Dashboard 2.0 sidebar.
+2. In the page configuration, you can select the preferred layout for that page within the layout field.
+
+!["Image showing process of setting page layout"](./images/node-red-dashboard-2-layout-navigation-styling-setting-new-page-layout.gif "Image showing process of setting page layout"){data-zoomable}
+
+## Setting Dashboard 2.0 elements size
+
+Setting the size for elements in Dashboard 2.0 is straightforward, but understanding the actual unit size in the size property can be a bit tricky. It's important to note that the size of a horizontal unit varies depending on the layout, while the vertical size is fixed at **48px** currently.
+
+## Understanding unit size in different layouts
+
+#### Grid
+
+In a grid layout, each unit specified in the size property represents a column, dividing the page evenly. When you assign columns to a group, it forms an internal grid with the same number of columns as the group. For example, if you have a group with 4 columns and two widgets inside it, each widget can be sized by considering those columns. If you set 1 column for the first widget and 3 columns for the second widget, the first widget will occupy 25% of the group's width, and the second widget will occupy 75%.
+
+#### Notebook
+
+In the Notebook layout, As you know the width for the group is fixed at **1024px**. The size unit represents the number of columns that the group will create or represent in its internal grid, increasing size will increase the number of columns in group. For example, if you have one group with a size of 12 columns, that group will create an internal grid with 12 columns, and widgets can occupy those columns using the size property where each will also represent one column in group.
+
+#### Fixed
+
+The fixed layout in Dashboard 2.0 employs a flexbox design with a single row of widgets. Each width unit in this layout is set at a fixed 90px. Groups follow the same internal grid pattern as other layouts. When a widget occupies the entire screen width, it shifts to the next row. However, this layout remains unchanged for varying screen sizes, potentially resulting in empty space on larger screens.
```

Comment:
This whole section explains how widgets are sized within a group, no need to differentiate between the 3 x layouts - just have a single "Sizing Widgets" section that explains how a group lays out widgets.